LogMonitor/src/LogMonitor/Main.cpp

```diff
@@ -198,6 +213,12 @@ void StartMonitors(_In_ LoggerSettings& settings)
 
 int __cdecl wmain(int argc, WCHAR *argv[])
 {
+    InitTracer();

```

Comment:
nit: could this be behind a feature flag so that we don't have to pay for it when a user doesn't want to report telemetry?

Reply:
Makes sense!

Reply:
In the meantime I'll have this as an entry in the config file. I'm trying to review the document I had to see if we agreed on something different but can't see it. 
@iankingori

Reply:
Traced it down to environment variable.